presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -367,10 +361,10 @@ public HttpRemoteTask(
                     initialTask.getTaskStatus(),
                     taskStatusRefreshMaxWait,
                     taskStatusCodec,
-                    executor,
+                    taskEventLoop,
                     httpClient,
                     maxErrorDuration,
-                    errorScheduledExecutor,
+                    taskEventLoop,
```

Comment:
Let's do the same exercise (removing synchronization) for the `ContinuousTaskStatusFetcher` and the `TaskInfoFetcher`